### PR TITLE
Uyuni 2024.03

### PR DESCRIPTION
**Pull request by raulillo82**
Uyuni 2024.03

### CODE REVIEW

**Review by juliogonzalez — APPROVED** [review]

**Review by deneb-alpha — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -233,26 +233,25 @@ <h3>Latest news</h3>
         <div class="js-home-carousel home-carousel">
 
           <!-- News box Start -->
-	  <div class="news-box">
+          <div class="news-box">
             <div class="news-box-title">
-              <span>21/02/2024</span>
+              <span>15/04/2024</span>
```
Comment (deneb-alpha) [inline index.html:238]:
```suggestion
              <span>16/04/2024</span>
```

File: pages/news.html [hunk 2]
```diff
@@ -88,7 +88,8 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
-            <p><strong>21/02/2024</strong> - Starting from February, the Uyuni Community Hours will take place on <a href="https://meet.jit.si/moderated/08658cae0e2eefa6e84ef2229a34e339e1b3b473449ae06e4610cfaad5a32b0b" target="_blank">Jitsi</a>. Check the <a href="https://lists.opensuse.org/archives/list/announce@lists.uyuni-project.org/message/7553PZDBOCC4EGXUPNFNBO73QWKPG5DK/" target="_blank">announcement</a> to get all the details!</a></p>
+            <p><strong>15/04/2024</strong> - Uyuni 2024.03 is now available with Confidential Computing Attestation, enhanced CVE Audit, Ansible security fixes and more! Check the <a href='../pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```
Comment (deneb-alpha) [inline pages/news.html:91]:
```suggestion
            <p><strong>16/04/2024</strong> - Uyuni 2024.03 is now available with Confidential Computing Attestation, enhanced CVE Audit, Ansible security fixes and more! Check the <a href='../pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
please update release date with the suggestions I made.
Please, also take the updated html for the release notes. 
the gitlab project with the release notes is updates and also OBS. but I can't pushed the refreshed html, you used a branch on your fork.

**Review by deneb-alpha — APPROVED** [review]
LGTM, but again,let's wait  for legal before merging

**Review by juliogonzalez — APPROVED** [review]

**Comment by juliogonzalez** [comment]
Could be you need to change the PR a little according to what happens with the Uyuni Server as containers migrations/updates

**Pull request merged**